index.html

```diff
@@ -0,0 +1,135 @@
+          <input type="text" placeholder="Search here..." />
+          <button>Search</button>
+        </div>
+        <div class="table" id="table1">
+          <h2 class="itemname">Table1</h2>
+          <p>Cost: Rs.<span id="table1-cost">0</span></p>
+          <p>Total Items: <span id="table1-count">0</span></p>
+        </div>
+        <div class="table" id="table2">
+          <h2 class="itemname">Table2</h2>
+          <p>Cost: Rs.<span id="table2-cost">0</span></p>
+          <p>Total Items: <span id="table2-count">0</span></p>
+        </div>
+        <div class="table" id="table3">
+          <h2 class="itemname">Table3</h2>
+          <p>Cost: Rs.<span id="table3-cost">0</span></p>
+          <p>Total Items: <span id="table3-count">0</span></p>
+        </div>
+        <div class="table" id="table4">
+          <h2 class="itemname">Table4</h2>
+          <p>Cost: Rs.<span id="table4-cost">0</span></p>
+          <p>Total Items: <span id="table4-count">0</span></p>
+        </div>
+      </div>
+      <div class="vl"></div>
+      <div class="menu">
+        <div class="title">
+          <h1>Menu</h1>
+        </div>
+        
+        <div class="menu-search" id="menu-search">
+          <input type="text" placeholder="Search here.." />
+          <button>Search</button>
+        </div>   
+        <div class="menusort"></div>
+        <div class="item">
+          <h2 class="itemname" draggable="true" id="masaladosa" foodtype="biryani">Chicken Dum Biryani</h2>
+          <p>Rs.250</p>
+        </div>
+        <hr>  
+        <div class="item">
+          <h2 class="itemname" draggable="true" id="oniondosa" foodtype="biryani">Chillie Chicken</h2>
+          <p>Rs.245</p>
+        </div> 
+        <hr>
+        <div class="item">
+          <h2 class="itemname" draggable="true" id="idly" foodtype="biryani">Chicken kebab</h2>
+          <p>Rs.135</p>
+        </div> 
+        <hr>
+        <div class="item">
+          <h2 class="itemname" draggable="true" id="chapathi" foodtype="biryani">Veg Biryani</h2>
+          <p>Rs.160</p>
+        </div> 
+        <hr>
+        <div class="item">
+          <h2 class="itemname" draggable="true" id="kinley" foodtype="starters">Dragon Chicken</h2>
+          <p>Rs.225</p>
+        </div> 
+        <hr>
+        <div class="item">
+          <h2 class="itemname" draggable="true" id="thumbsup" foodtype="starters">Chicken 65</h2>
+          <p>Rs.245</p>
+        </div> 
+        <hr>
+        <div class="item">
+          <h2 class="itemname" draggable="true" id="cococola" foodtype="biryani">Special Chicken Biryani</h2>
+          <p>Rs.285</p>
+        </div>
+        <hr>
+        <div class="item">
+          <h2 class="itemname" draggable="true" id="frenchfries" foodtype="fries">French Fries</h2>
+          <p>Rs.145</p>
+        </div>
+        <hr>
+        <div class="item">
+          <h2 class="itemname" draggable="true" id="vanillaicecream" foodtype="icecream">Vannila Icecream</h2>
```

Comment:
As discussed these need to be taken from json file right?

Reply:
Yeah i tried but i didnt getting the output. so kept like default only in stand up they said ok so moved on